models/hashflow/avalanche_c/hashflow_avalanche_c_crosschain_trades.sql

```diff
@@ -0,0 +1,103 @@
+{{
+    config(
+        alias="cross_chain_trades"
+        ,partition_by = ['block_date']
+        ,materialized='incremental'
+        ,incremental_strategy = 'merge'
+        ,unique_key = ['block_date', 'source_chain', 'tx_hash']
+        ,post_hook='{{ expose_spells(\'["avalanche_c"]\',
+                                        "project",
+                                        "hashflow",
+                                        \'["BroderickBonelli"]\') }}'
+    )
+}}
+
+
+with cross_chain_trades AS (
+        SELECT
+            evt_block_time           AS block_time
+            ,trader                  
+            ,quoteTokenAmount        AS token_bought_amount_raw
+            ,baseTokenAmount         AS token_sold_amount_raw
+            ,cast(NULL AS double)    AS amount_usd
+            ,quoteToken              AS token_bought_address
+            ,baseToken               AS token_sold_address
+            ,evt_tx_hash             AS tx_hash
+            ,CASE WHEN dstChainId = 1 OR dstChainId = 101 THEN 'Ethereum'
```

Comment:
@BroderickBonelli i know you guys mentioned creating these spells separate, which is fine, but i think it would still make sense to follow the standard of lowercase for hardcoded values such as blockchain. in case you ever use these spells to join to another spell which uses that approach. can you make these all lowercase?

Reply:
Yep no problem, will do